python/common/org/python/types/Range.java

```diff
@@ -118,4 +121,50 @@ public Range(org.python.Object start, org.python.Object stop, org.python.Object
         }
     }
 
+    @org.python.Method(
+        __doc__ = "Implement __len__(self)."
+    )
+    public org.python.types.Int __len__() {
```

Comment:
Yes, it should - Java will actually let you get away with this, because the return type doesn't formally form part of the method prototype, but for consistency, it should return `org.python.Object`.